scripts/build.sh

```diff
@@ -46,6 +46,12 @@ install_adu=false
 work_folder=/tmp
 cmake_dir_path="${work_folder}/deviceupdate-cmake"
 
+#
+# Export the compiler settings in case VM is wonky
```

Comment:
What would be the common or expected errors when VM is "wonky"?

Reply:
CC/ CXX is not found seems to be an intermittent issue on some of these virtual machines.

The true fix would be to add a 1ES artifact that sets up and exports these variables in a programmatic way.